### ISSUE TITLE
get labix.org/v2/mgo error The remote server unexpectedly closed the connection

### ISSUE THREAD

**Issue by Yang09701194** [arschles/go-in-5-minutes#149]
I'm trying to run a main.go file in episode5, and it shows error:

```
cannot find package "labix.org/v2/mgo" in any of:
    C:\Go\src\labix.org\v2\mgo (from $GOROOT)
    D:\MyFolder\go-in-5-minutes\episode5\src\labix.org\v2\mgo (from $GOPATH)
cannot find package "labix.org/v2/mgo/bson" in any of:
    C:\Go\src\labix.org\v2\mgo\bson (from $GOROOT)
    D:\MyFolder\go-in-5-minutes\episode5\src\labix.org\v2\mgo\bson (from $GOPATH)
exit status 1     Process exiting with code: 1
```
so I try to execute  

`D:\MyFolder\go-in-5-minutes\episode5\> go get labix.org/v2/mgo`

But then I got this error:
```
:cd .; bzr branch https://launchpad.net/mgo/v2 D:\MyFolder\go-i n-5-
minutes\episode5\src\labix.org\v2\mgo bzr: 
ERROR: The remote server unexpectedly closed the connection. package labix.org/v2/mgo: 
exit status 3
```

How to solve this problem and get labix.org/v2/mgo ?
<br/>
Besides, someone indicate that `labix.org/v2/mgo` has been moved to `gopkg.in/mgo.v2`, is that right?

(I ask the same question [HERE](https://stackoverflow.com/questions/48921293/get-labix-org-v2-mgo-error-the-remote-server-unexpectedly-closed-the-connection))

**Comment by Yang09701194** [comment]
Currently I replace all the files using

```
import (
	"labix.org/v2/mgo"
	"labix.org/v2/mgo/bson"
)
```

with

```
import{
    "gopkg.in/mgo.v2"
    "gopkg.in/mgo.v2/bson"
}
```

and pass build and works!

**Comment by arschles** [comment]
Hi @Yang09701194 - great catch, thank you for filing this issue 😄 

Would you mind submitting a pull request with the fix you put above?

**Comment by Yang09701194** [comment]
Thank you! I can submit a pull request with this. 

This is the first time I have a chance to make a pull request on github , should I make a pull request directly at master branch, or create a subbranch?

**Comment by arschles** [comment]
Great, thank you very much!

To make the pull request, fork this repository into your own and then submit from your own repository's branch directly onto the master branch in this repository.

Here's some additional documentation if you'd like to read: https://help.github.com/articles/creating-a-pull-request/. Apologies if you've already read it 😄

**Comment by Yang09701194** [comment]
Thanks for your guiding and approve! I'm willing to see this issue be closed.

**Comment by arschles** [comment]
absolutely. I'm really, really happy I could help you submit and merge your first PR!

for the next time, you can put `closes #<your PR number>` into your pull request and when it's merged, the issue will be automatically closed.

so, for example, you could put the following into your PR (#150): `closes #149`. I love that feature in Github. check it out next time :)
